pcdsdevices/interface.py

```diff
@@ -844,16 +844,21 @@ def mv_ginput(self, timeout=None):
         except ophyd.utils.LimitError:
             return
 
-    def tweak(self):
+    def tweak(self, **kwargs):
```

Comment:
The only suggestion I have is to switch this to the much simpler `scale=0.1` in the signature here so that the user can more easily see the arguments when they inspect the function. I realize that this is a duplication of the default value, but I think the API clarity is more important.